### ISSUE TITLE
Cannot insert publication year and authors

### ISSUE THREAD

**Issue by Crrreature** [vanakat/zotero-link#6]
For example, when I use `{{title}}` as template, I can get

```
[Jing et al_2020_Community Detection on Mixture Multi-layer Networks via Regularized Tensor.pdf](zotero://select/library/items/RWFDKJEA)
```

However, after changing the template to `{{ title }}, {{ date.year }}, {% for author in authors %}{{ author.lastName }}, {{ author.firstName | first }}., {% endfor %}}`, what I got is:

```
[Jing et al_2020_Community Detection on Mixture Multi-layer Networks via Regularized Tensor.pdf, , }](zotero://select/library/items/RWFDKJEA)
```

It seemed that the publication date and authors cannot be resolved by the plugin. Are there any way to fix it ?

**Comment by GTTull** [comment]
I have the same issue!!

**Comment by MunGell** [comment]
Hi,

Sorry for the delay in response, I was looking into this issue now and I think I have a hypothesis of what is happening there.

The underlying plugin (and underlying Zotero API) doesn't let me distinguish between parent items and files, so at the moment they all appear in a flat list (related to https://github.com/vanakat/zotero-bridge/issues/5).

In your particular case you are linking to a file, which doesn't have metadata such as author and date.

Could you please check if you could create a link to a parent item instead (should have no .pdf in the end of the title) and if it resolves your issue?

**Comment by MunGell** [comment]
The issue should be solved now in Zotero Bridge v1.3.0

Please let me know if you still experience it!
